mypy.ini

```
@@ -17,7 +17,7 @@ txt_report = mypy
 ; TODO: Adopt --strict settings, iterating towards something like:
 ; https://github.com/pypa/packaging/blob/master/setup.cfg
 ; Starting with modules that have annotations applied via MonkeyType
-[mypy-twine.auth,twine.cli,twine.exceptions,twine.package,twine.repository,twine.utils,twine.commands,twine.wheel,twine.wininst]
+[mypy-twine.auth,twine.cli,twine.exceptions,twine.package,twine.repository,twine.utils,twine.commands,twine.wheel,twine.wininst,twine.command.register]
```

Typo in the mypy config:

```suggestion
[mypy-twine.auth,twine.cli,twine.exceptions,twine.package,twine.repository,twine.utils,twine.commands,twine.wheel,twine.wininst,twine.commands.register]
```

So, `twine.commands.register` hasn't been checked with the strict settings.

---

Fixed in latest commit. (Good that the annotations still worked without any changes after correcting the submodule name 😊 )